scss/sidebar.scss

```diff
@@ -0,0 +1,102 @@
+// Sidebar Style Overrides
+.sidebar-wrapper {
+  background: none;
+
+  .sidebar-scroll-wrap {
+    --scrollbarThumbBg: var(--primary-300);
+    --scrollbarWidth: 0.5em;
+
+    &:hover {
+      &::-webkit-scrollbar-thumb {
+        border: none;
+      }
+    }
+  }
+}
+
+.sidebar-section-header.btn {
+  border: none;
+}
+
+.sidebar-footer-wrapper {
+  background: none;
+  .desktop-view & {
+    .sidebar-footer-container {
+      background: none;
+      &:before {
+        background: linear-gradient(
+          to bottom,
+          transparent,
+          rgba(var(--primary-low-rgb), 1)
+        );
+      }
+    }
+  }
+}
+
+// Fakebook overrides when sidebar_enabled
+.has-sidebar-page {
+  @media screen and (min-width: 650px) {
+    &.navigation-topics,
+    &.categories-list,
+    &.tags-page,
+    &[class*="category-"]:not(.archetype-regular):not(.archetype-banner) {
```

Comment:
That is quite a selector! It is targeting a subset of category landing pages?

Reply:
Yes, I believe so. It was used previously in the theme (`common.scss`), I simply needed to add the `&.` because I am wrapping the selector in the `has-sidebar-page` class so that the changes are only applied for the sidebar. Maintaining the same selector so we don't have specificity issues.